conwaywebclienttests/WebClientTests.cs

```diff
@@ -0,0 +1,64 @@
+using NUnit.Framework;
+using ConwayWebClient;
+using System.Net.Http;
+using ConwayLib;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace ConwayWebClientTests
+{
+    [TestFixture]
```

Comment:
You were going to make this Explicit. It definitely won't pass on the build agent

Reply:
Now done, I see